ext/droption/droption.h

```diff
@@ -453,6 +453,17 @@ droption_t<unsigned int>::convert_from_string(const std::string s)
     return true;
 }
 template<> inline bool
+droption_t<double>::convert_from_string(const std::string s) {
+  double input = atof(s.c_str());
```

Comment:
style: indentation throughout this function

Reply:
Done.